### PR TITLE
The Body Archive now restores the original status of limbs and internal organs

### PR DESCRIPTION
**Pull request by DeityLink**
image

Behold my army of bootleg Athens

### CODE REVIEW

File: code/datums/body_archive.dm [hunk 1]
```diff
@@ -115,7 +115,32 @@ var/list/body_archives = list()
 	archive.data["dna_records"] = R
 	archive.data["underwear"] = underwear
 
-	//I gave a shot at preserving limb status (destroyed, peg, mechanized) then decided I was too tired and it's too much trouble, so TODO for anyone with the determination
```
Comment (DeityLink) [inline code/datums/body_archive.dm:118]: There's something to be learned here but I'm not sure what it was.

Comment (Eneocho) [reply code/datums/body_archive.dm:118]: not coding while tired?

Comment (DeityLink) [reply code/datums/body_archive.dm:118]: maybe that yeah

**Comment by Hinaichigo** [comment]
Does this apply to cloning from backup? If so then maybe it makes sense to have limbs restored even if they were missing at roundstart.

**Comment by DeityLink** [comment]
The Body Archive is an entirely separate feature from cloning. It's an admin tool. If admins want to restore a player's limbs they have another button for that.

**Pull request merged**